nautobot/extras/templates/extras/inc/job_tiles.html

```diff
@@ -0,0 +1,72 @@
+{% load helpers %}
+
+{% block extra_styles %}
+    <style>
+        .panel.panel-default {
```

Comment:
Where is this style being used in this template?

Reply:
Job list content is wrapped in `panel panel-default` (via `panel_table.html`) element which works for the default job table display. In case of tiles, though, we don't want any borders around them and that's essentially what panels do, so I override those styles here. If there is a better way to fix this, I'll be happy to make it right. Here's how tiles display would look like without overriding panel `border` and `box-shadow`:
![panel_border](https://github.com/nautobot/nautobot/assets/117445994/077ab005-67f0-4668-bbf7-0ddc3469bad7)